onnxscript/rewriter/pattern_test.py

```diff
@@ -57,6 +59,16 @@ def test_failed_match(self):
         self.assertEqual(count, 0)
         self.assertEqual(len(model.graph), 4)
 
+        # Test verbose output produces something:
+        # TODO(rama): Need a better way to test this.
+        # Well-defined error-codes and messages would be helpful.
+
+        buffer = io.StringIO()
+        with contextlib.redirect_stdout(buffer):
+            self.rule().apply_to_model(model, verbose=5)
```

Comment:
Possible to specify verbosity levels and standardize it?

Reply:
Yes, that would be useful. For now, I am just doing something similar to Xavier's implementation, so that this doesn't become a reason to choose one implementation over another. I am happy to leave it to Xavier, as the primary user who uses this, to flesh this out depending on his usage experience.

Reply:
SG. Thanks!